borg/archiver.py

```diff
@@ -1153,6 +1153,13 @@ def build_parser(self, args=None, prog=None):
         browsing an archive or restoring individual files. Unless the ``--foreground``
         option is given the command will run in the background until the filesystem
         is ``umounted``.
+
+        The command ``borgfs`` provides a wrapper for ``borg mount``. This can also be
+        used in fstab entries:
+        ``/path/to/repo::archive /mnt/point fuse.borgfs defaults,noauto 0 0``
+
+        To allow a regular user to use fstab entries, add the ``user`` option:
+        ``/path/to/repo::archive /mnt/point fuse.borgfs defaults,noauto,user 0 0``
```

Comment:
maybe we should rather just have the repo here without archive specification. the archive name changes all the time, so it is unlikely somebody would use that in fstab.